pkgs/applications/networking/browsers/chromium/default.nix

```diff
@@ -3,6 +3,7 @@
 , glib, gtk3, gnome, gsettings-desktop-schemas, gn, fetchgit
 , libva, pipewire, wayland
 , gcc, nspr, nss, runCommand
+, fontconfig, roboto, twitter-color-emoji, makeFontsConf
```

Comment:
Hm, those two fonts (roboto and twitter-color-emoji) seem kind of arbitrary to me. Why them? What if a website uses some other font? I'm afraid the solution doesn't scale well.

Reply:
@NorfairKing can probably answer that.

Reply:
I just chose some fonts that would make emoji show up :shrug:

Reply:
@NorfairKing But this is because Chromium crashes headlessly without emojis in your tests, right?

Reply:
@MatthewCroughan because I was doing golden tests of screenshots that included emoji.

Reply:
That's what I thought :) So basically the above approach doesn't work in general because you can't possibly list *all* available fonts.